modin/engines/ray/pandas_on_ray/sql.py

```diff
@@ -0,0 +1,206 @@
+from collections import OrderedDict
+from sqlalchemy import MetaData, Table, create_engine
+
+
+def is_distributed(partition_column, lower_bound, upper_bound):
+    """ Check if is possible distribute a query given that args
+
+    Args:
+        partition_column: column used to share the data between the workers
+        lower_bound: the minimum value to be requested from the partition_column
+        upper_bound: the maximum value to be requested from the partition_column
+
+    Returns:
+        True for distributed or False if not
+    """
+    if (
+        (partition_column != None) and (lower_bound != None) and (upper_bound != None)
```

Comment:
For `None` I think the linter prefers `is not` instead of `!=`. Then you can get rid of the `noqa`.

Reply:
fixed